cssdb.json

```
@@ -373,6 +373,15 @@
       }
     ]
   },
+  {
+    "id": "in-out-of-range",
```

Could we add the `pseudo-class` suffix to the ID, matching other pseudo classes?

```suggestion
    "id": "in-out-of-range-pseudo-class",
```